gunicorn/arbiter.py

```diff
@@ -491,20 +491,19 @@ def murder_workers(self):
         """
         if not self.timeout:
             return
-        workers = list(self.WORKERS.items())
-        for (pid, worker) in workers:
+        for pid, worker in self.WORKERS.items():
             try:
                 if time.time() - worker.tmp.last_update() <= self.timeout:
                     continue
             except (OSError, ValueError):
                 continue
 
-            if not worker.aborted:
+            if worker.aborted:
```

Comment:
I always prefer positive checks over negative checks, but if you prefer not to change it let's keep it.

Reply:
well it's maybe an overoptimisation but since the case `not aborted` is the normal case it's more natural and maybe  faster  to handle that case first. Then yes let's keep it that way then